platform/debian/kanidm-unixd/rules

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/make -f
+# See debhelper(7) (uncomment to enable)
+
+# be REALLY noisy
+export DH_VERBOSE=1
+export DH_OPTIONS=-v
+
+export DEB_BUILD_OPTIONS=noautodbgsym
+PACKAGE=kanidm-unixd
+PKGDIR=debian/${PACKAGE}
+BINDIR=${PKGDIR}/usr/sbin/
+SHARED_DIR=${PKGDIR}/usr/share/${PACKAGE}
+
+%:
+	dh $@
+
+override_dh_auto_clean:
+
+override_dh_autoreconf:
+
+override_dh_auto_build:
+	dh_auto_build -- release/${PACKAGE}
```

Comment:
Is this relying on the make release/ from above?

Reply:
yep, the dh tooling assumes makefiles in the build process